packages/aws-cdk/lib/api/hotswap/common.ts

```diff
@@ -98,6 +98,52 @@ export class HotswappableChangeCandidate {
 
 type Exclude = { [key: string]: Exclude | true }
 
+/**
+ * Represents configuration properties for hotswap deployments
+ */
+export class HotswapProperties {
```

Comment:
```suggestion
export class HotswapPropertyOverrides {
```

Reply:
I have renamed the class and also all the variables as we propagate the properties throughout the code path to reflect the new name.